Formula/c/cowsay.rb

```diff
@@ -22,7 +22,7 @@ class Cowsay < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "8d35c9dfb46eea22b2b53c9c0deb00d7d95b6fe3fcfeb8d9404fd269d5739790"
   end
 
-  deprecate! date: "2023-11-22", because: :repo_archived
+  disable! date: "2024-09-09", because: :repo_archived
```

Comment:
```suggestion
  disable! date: "2024-11-22", because: :repo_archived
```
Given that `brew info cowsay` says
```
❯ brew info cowsay
==> cowsay: stable 3.04 (bottled)
Configurable talking characters in ASCII art
https://github.com/tnalpgge/rank-amateur-cowsay
Deprecated because it has an archived upstream repository! It will be disabled on 2024-11-22.
```

Reply:
Our doc says six months, for > 1000 downloads in the last 90 days: https://docs.brew.sh/Deprecating-Disabling-and-Removing-Formulae#disabling

```
Popular formulae (e.g. have more than 1000 [analytics installs in the last 90 days](https://formulae.brew.sh/analytics/install/90d/)) should not be disabled without a deprecation period of at least six months even if e.g. they do not build from source and do not have a license.
```

It looks like the warning message is not aligned with our doc.

Reply:
It does say _at least_ six months.

In any case, we've already shown the message to users so may as well stick by it. Doesn't hurt to keep it around for a bit longer.